e2e2/test/cases/neuron/manifests/single_node_test_neuronx.yaml

```diff
@@ -0,0 +1,31 @@
+kind: Job
+apiVersion: batch/v1
+metadata:
+  name: neuronx-single-node
+  labels:
+    app: neuronx-single-node
+spec:
+  template:
+    metadata:
+      labels:
+        app: neuronx-single-node
+    spec:
+      containers:
+      - name: neuronx-single-node-test
+        image: {{.NeuronSingleNodeTestImage}}
+        args:
+        - torchrun --nproc_per_node=2 --nnodes=1 pytorch_tests/testNeuronSingleAllReduce &&
+        - torchrun --nproc_per_node=2 --nnodes=1 pytorch_tests/testNeuronParallelState &&
+        - torchrun --nproc_per_node=2 --nnodes=1 pytorch_tests/testNeuronMlp
```

Comment:
This looks pretty odd, can we move this into a shell script in the container or something?

Reply:
Fixed